app/models/user.rb

```diff
@@ -337,4 +337,8 @@ def favorite_collections_for_project(project)
   def subjects_count_cache_key
     @subjects_count_cache_key ||= "User/#{id}/uploaded_subjects_count"
   end
+
+  def subject_count_cache_expiry
+    ENV.fetch("UPLOADED_SUBJECTS_COUNT_CACHE_EXPIRY", 1.hour)
```

Comment:
the env file that docker-compose exposes to the container at runtime, it's in prod configs.